test/UtilityFunctionsTests.cpp

```diff
@@ -70,3 +70,24 @@ SCENARIO("check sumNumbersFromGenerators() function")
         }
     }
 }
+
+SCENARIO("check sumNumbersFromGenerators() function with Not existing generator type")
+{
+    GIVEN("Empty generators collection and numberOfValues = 1")
+    {
+        auto generators = {(int)GeneratorType::FIBONACCI,
+                           -1,
+                           (int)GeneratorType::PRIME};
```

Comment:
generators is an `std::initializer_list<int>`. It would be easier to have something like this:
```cpp
 std::vector<GeneratorType> generators = {GeneratorType::FIBONACCI,
                                          static_cast<GeneratorType>(-1),
                                          GeneratorType::PRIME};
```

Reply:
Right, this is easier.
Code updated